Common/Brokerages/GDAXBrokerageModel.cs

```diff
@@ -33,23 +33,23 @@ public class GDAXBrokerageModel : DefaultBrokerageModel
         // https://support.gdax.com/customer/portal/articles/2725970-trading-rules
         private static readonly Dictionary<string, decimal> MinimumOrderSizes = new Dictionary<string, decimal>
         {
-            { "BTCUSD", 0.0001m },
-            { "BTCEUR", 0.0001m },
-            { "BTCGBP", 0.0001m },
-
-            { "BCHUSD", 0.0001m },
-            { "BCHEUR", 0.0001m },
-            { "BCHBTC", 0.0001m },
-
-            { "ETHUSD", 0.001m },
-            { "ETHEUR", 0.001m },
-            { "ETHGBP", 0.001m },
-            { "ETHBTC", 0.001m },
-
-            { "LTCUSD", 0.01m },
-            { "LTCEUR", 0.01m },
-            { "LTCGBP", 0.01m },
-            { "LTCBTC", 0.01m }
+            { "BTCUSD", 0.001m },
+            { "BTCEUR", 0.001m },
+            { "BTCGBP", 0.001m },
+
+            { "BCHUSD", 0.01m },
+            { "BCHEUR", 0.01m },
+            { "BCHBTC", 0.01m },
+
+            { "ETHUSD", 0.01m },
+            { "ETHEUR", 0.01m },
+            { "ETHGBP", 0.01m },
+            { "ETHBTC", 0.01m },
```

Comment:
Limit Orders:
The line you quoted appears to be wrong (it was 0.001 in the previous update though), minimum limit order size for ETHBTC is 0.01, I tested all minimum quantities for limit orders with the GDAX platform manually.

Market Orders:
I believe the quantities shown in quote currency are for platform use (or API code if using the _funds_ field instead of _size_ -- currently not supported by LEAN) where you must specify an amount in quote currency instead of base currency (e.g. _I want to buy 10000 USD worth of BTC_ vs _buy 1 BTC_)

Reply:
Interesting. I wasn't aware of the funds size field.